### PR TITLE
Implement proper encoding name normalizaton (close #554)

### LINKED ISSUE #554: Implement proper encoding name normalizaton
Currently we have pretty trivial encoding name normalization: https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/processing/encoding/charset.js#L50

But possible encoding aliases (aka labels) are defined in HTML spec: https://encoding.spec.whatwg.org/#names-and-labels
We need use this labels table instead.

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @LavrovArtem

### CODE REVIEW

File: src/processing/encoding/labels.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    'big5':           ['big5', 'big5-hkscs', 'cn-big5', 'csbig5', 'x-x-big5'],
+    'euc-jp':         ['cseucpkdfmtjapanese', 'euc-jp', 'x-euc-jp'],
+    'iso-2022-jp':    ['csiso2022jp', 'iso-2022-jp'],
+    'shift_jis':      ['csshiftjis', 'ms932', 'ms_kanji', 'shift-jis', 'shift_jis', 'sjis', 'windows-31j', 'x-sjis'],
+    'euc-kr':         ['cseuckr', 'csksc56011987', 'euc-kr', 'iso-ir-149', 'korean', 'ks_c_5601-1987', 'ks_c_5601-1989', 'ksc5601', 'ksc_5601', 'windows-949'],
+    'replacement':    ['csiso2022kr', 'hz-gb-2312', 'iso-2022-cn', 'iso-2022-cn-ext', 'iso-2022-kr'],
+    'utf-16be':       ['utf-16be'],
+    'utf-16le':       ['utf-16', 'utf-16le'],
+    'x-user-defined': ['x-user-defined']
+};
```
Comment (inikulin) [inline src/processing/encoding/labels.js]: We can build lookup table from this one:
```js
var LOOKUP_TABLE = ENCODING_LABELS.reduce((table, encoding) => {
     ENCODING_LABELS[encoding].forEach(label => table[label] = encoding);
     return table;
}, {});
```

Then we will have O(1) lookups:
```js
return LOOKUP_TABLE[label];
```

**Comment by inikulin** [comment]
\r-

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Comment by churkin** [comment]
@inikulin ping

**Comment by inikulin** [comment]
lgtm

**Pull request merged**